### PR TITLE
add loading animation

### PR DESCRIPTION
**Pull request by cjbt**
# Description ✔️ 

Adds loading animation which fixes the loginWithRedirect() is undefined error

## Checklist 👍 

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "react": "^16.8.6",
     "react-conflux": "^0.0.2-beta.6",
     "react-dom": "^16.8.6",
+    "react-loader-spinner": "^3.1.2",
```
Comment (BrannanC) [inline client/package.json]: Aw yea

File: client/src/components/Layout/Footer.jsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import styled from 'styled-components';
 
-const Footer = props => {
+const Footer = () => {
```
Comment (BrannanC) [inline client/src/components/Layout/Footer.jsx]: Should I give you props for this :joebacus:

File: client/package.json (hunk 1 above)
Comment (nwthomas) [reply client/package.json]: 🔥🔥🔥

File: client/src/components/Layout/Footer.jsx (hunk 2 above)
Comment (nwthomas) [reply client/src/components/Layout/Footer.jsx]: 🔥🔥🔥

**Pull request merged**